CODEOWNERS

```diff
@@ -1,2 +1,11 @@
-CODEOWNERS @guidanoli
+# ABOUT CODE OWNERS
+# THIS FILE FOLLOWS https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
```

Comment:
I would remove these lines.
If a developer is not sure what `CODEOWNERS` is for, they can just Google it.
In fact, this page is the first thing that pops when you query Google about `CODEOWNERS`.

Reply:
Can agree in this one.